src/components/asset-item.tsx

```diff
@@ -58,10 +58,10 @@ export const AssetItem = memo(
                 <Box>
                   <Tooltip
                     placement="left-start"
-                    label={formatted.isAbbreviated ? amount : undefined}
+                    label={formatted?.isAbbreviated ? amount : undefined}
                   >
                     <Text fontVariantNumeric="tabular-nums" textAlign="right">
-                      {formatted.value}
+                      {formatted?.value}
```

Comment:
what caused this to be potentially undefined? this should probably just never render if the value is null/undefined

Reply:
This comes from the `stxTokenState` possibly being `undefined` but I'll look at fixing it.